scss/core/base.scss

```diff
@@ -10,37 +10,65 @@ TABLES
 
 /* LINKS */
 a {
+  font-family: $linkFontFamily;
+  font-size: $linkFontSize;
   color: $linkColor;
-  text-decoration: none;
+  font-weight: $linkFontWeight;
+  font-style: $linkFontStyle;
+  text-decoration: $linkFontDecoration;
 
   &:hover,
   &:focus,
   &:active:hover,
   &:active:focus {
+    font-family: $linkFontFamilyHover;
```

Comment:
For the hover and focus selectors, if we only keep `color` and `outline`, will that work? If not, what's overriding these properties?

Reply:
Yes, it will work. Should I leave in hover/focus only color with variable and `outline: none`?

Reply:
@ivandevupp Yes, please. This way the generated CSS file is smaller.

Reply:
Probably didn't get right your question, all options selected for text will work, but we need to customize text hover too 
![image](https://user-images.githubusercontent.com/74735207/104749264-44a16380-575b-11eb-9479-8030b71ae614.png)

Reply:
I see. Looks fine then.